lib/elixir_tic_tac_toe_basic/ui.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ElixirTicTacToeBasic.UI do
+  @moduledoc """
+  Provides a user interface.
+  """
+
+  def welcome(state, callback \\ fn m -> IO.puts(m) end) do
```

Comment:
**note**: Karen and I discussed the option to push this `callback` into the state hash at some point.